src/routes/safe/components/Transactions/TxList/styled.tsx

```diff
@@ -378,24 +384,23 @@ export const TxDetailsContainer = styled.div`
 
   .tx-owners {
     padding: 24px;
-    grid-column-start: 2;
+    /* grid-column-start: 2;
+    grid-row-start: 1; */
     grid-row-end: span 2;
-    grid-row-start: 1;
 
     &.no-owner {
-      grid-row-end: span 3;
+      /* grid-row-end: span 3; */
```

Comment:
Have you tested this? This would be unstyled.

Reply:
I've tested and removed the class as it became obsolete.